tutorials/34_Model_Based_Evaluation_of_RAG_Pipelines.ipynb

```diff
@@ -0,0 +1,4480 @@
+{
```

Comment:
<p>Let's add the OpenAI api key page to "Prerequisites" :<a href="https://platform.openai.com/api-keys" target="_blank">https://platform.openai.com/api-keys</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done